docs/core/build/distribution-packaging.md

```diff
@@ -66,7 +66,7 @@ While there's a single host, most of the other components are in versioned direc
 
 - (3) **sdk/\<sdk version>** The SDK (also known as "the tooling") is a set of managed tools that are used to write and build .NET Core libraries and applications. The SDK includes the .NET Core Command-line interface (CLI), the managed languages compilers, MSBuild, and associated build tasks and targets, NuGet, new project templates, and so on.
 
-- (4) **sdk/NuGetFallbackFolder** contains a cache of NuGet packages used by an SDK during the restore operation, such as when running `dotnet restore` or `dotnet build /t:Restore`. This folder is only used prior to .NET Core 3.0. It can't be built from source, because it contains prebuilt binary assets from `nuget.org`.
```

Comment:
`dotnet build` also calls restore, but in a safe way. adding `-t:Restore` would only run it again but it isn't safe due to MSBuild's file caches and restore potentially overwriting files. The only safe way to run restore during build is by passing `-restore` (`-r` short)